hubgrep/frontend_blueprint/routes/search.py

```diff
@@ -49,15 +54,17 @@ def search():
                 exclude_empty=form.exclude_empty,
                 created_after=form.created_after_dt,
                 created_before=form.created_before_dt,
-                updated_after=form.updated_after_dt
+                updated_after=form.updated_after_dt,
+                pushed_after=form.pushed_after_dt
             )
         except UserError as e:
             results = []
             user_errors.append(e)
+        time_search = time.time() - time_before
 
         results_paginated = results[results_offset:(results_offset + results_per_page)]
         pagination_links = get_page_links(request.full_path, results_offset, results_per_page, len(results))
-        search_feedback = get_search_feedback(len(results))
+        search_feedback = get_search_feedback(total_found, time_search)
```

Comment:
oh, I thought this time came from the search meta; is there no value, or not usable for some reason?

Reply:
i didnt check - whats interesting from user perspective is overall time, no?

Reply:
yep, but for debugging and dev, maybe both values are interesting (if there is a time value from meta at all, that is)